docs/design/autoware-interfaces/index.md

```diff
@@ -162,46 +162,8 @@ Currently, there is no required header.
 
 The interfaces whose communication method is Function Call use a common response status to unify the error format.
 For those interfaces, include the ResponseStatus shown below in the response with the name status.
```

Comment:
> include the ResponseStatus shown below in the response with the name status

Unfortunately, this sentence does not make sense. Even if the text that is being removed were to be left in, I see no `name` status. Could you please clarify the intended meaning here please?